Cargo.toml

```diff
@@ -11,11 +11,12 @@ documentation = "https://docs.rs/niffler"
 edition = "2018"
 
 [features]
-default = ["bz2", "lzma", "gz", "bgz"]
+default = ["bz2", "lzma", "xz", "gz", "bgz"]
```

Comment:
I didn't really mean that an entire new feature was needed, just an extra variant here https://docs.rs/niffler/2.3.1/niffler/basic/compression/enum.Format.html#variants

But other people can weigh in also.

Reply:
The only way to get an alias in enum is implement a const like this https://github.com/luizirber/niffler/blob/176893f27187018114fa8313b6a41898edbfd71c/src/basic/compression.rs#L29

I add the feature just to be coherent with other format.  I should add a documentation about this.